lib/Settings/About.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:url_launcher/url_launcher.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+
+class About extends StatelessWidget {
+  final controller = ScrollController();
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        centerTitle: true,
+        title: Text('About'),
+      ),
+      body: FutureBuilder(
+          future: rootBundle.loadString("translation/About/en-US.md"),
+          builder: (BuildContext context, AsyncSnapshot<String> snapshot) {
+            if (snapshot.hasData) {
+              return Markdown(
+                data: snapshot.data,
+                onTapLink: (text, url, title) {
+                  launch(url);
+                },
+              );
+            }
+
+            return Center(
+              child: CircularProgressIndicator(),
+            );
+          }),
```

Comment:
What's the `futureBuilder` used for? Just wondering!

Reply:
futureBuilder is similar to async function I think. Since the process of fetching markdown file is something happens simultaneously and not sure if it could happen, we need to use it to return an error if the file is not fetched. The `CircularProgressIndicator()` is to display loading signal at the same time while the markdown file is being fetched.
This is how I understand. Not sure if I understand or explain it correctly.

Reply:
Ohh, yeahh! That makes sense - I remember `async` functions from long ago!

I did some digging and the `flutter_markdown` package doesn't reference `Future`s very much, but are we specifically using a `FutureBuilder`, because we're rendering a file?

Reply:
Good question. Since the `loadString` method is in Future type, I think we do need `FutureBuilder`. Otherwise, I don't know how to load markdown file w/o `loadString`

Reply:
Cool, that makes sense - thanks for letting me know about that!